mlfinlab/labeling/excess_over_mean.py

```diff
@@ -0,0 +1,41 @@
+"""
+Return in excess of mean method
+
+Chapter 5.5, Machine Learning for Factor Investing, by Coqueret and Guida, (2020).
+"""
+import numpy as np
+
+
+def excess_over_mean(prices, binary=False):
+    """
+    Return in excess of mean labelling method. Sourced from "The benefits of tree-based models for stock selection"
```

Comment:
Was it sourced from Zhu er al? or did you copy paste and forget to change this? I see above you say its from Factor Investing... Which you don't specify in the median code.